### PR TITLE
Prepare 1.0.0

### PR DESCRIPTION
**Pull request by matt-edmondson**

### CODE REVIEW

File: ImGuiStyler/Alignment.cs [hunk 1]
```diff
@@ -4,14 +4,29 @@ namespace ktsu.ImGuiStyler;
 using ImGuiNET;
 using ktsu.ScopedAction;
 
+/// <summary>
+/// Provides methods for aligning content within a container.
+/// </summary>
 public static class Alignment
 {
+	/// <summary>
+	/// Centers the content within the available content region.
```
Comment (Damon3000s) [inline ImGuiStyler/Alignment.cs:13]: Should this mention that the cursor will be advanced to the next line?

File: ImGuiStyler/Alignment.cs [hunk 2]
```diff
@@ -4,14 +4,29 @@ namespace ktsu.ImGuiStyler;
...
+	/// </summary>
+	/// <param name="contentSize">The size of the content to be centered.</param>
 	public class Center(Vector2 contentSize) : CenterWithin(contentSize, new Vector2(ImGui.GetContentRegionAvail().X, contentSize.Y))
 	{
 	}
 
+	/// <summary>
+	/// Centers the content within a specified container size.
```
Comment (Damon3000s) [inline ImGuiStyler/Alignment.cs:21]: Should this mention how the cursor is advanced?

File: ImGuiStyler/Alignment.cs (hunk 1 above)
Comment (matt-edmondson) [reply ImGuiStyler/Alignment.cs:13]: I dont think its super nescessary as advancing to the next line is the default imgui behavior

File: ImGuiStyler/Alignment.cs (hunk 2 above)
Comment (matt-edmondson) [reply ImGuiStyler/Alignment.cs:21]: See: https://github.com/ktsu-dev/ImGuiStyler/pull/68#discussion_r1879304508

**Pull request merged**